Packs/CommonScripts/Scripts/ListUsedDockersImages/ListUsedDockersImages.py

```diff
@@ -0,0 +1,191 @@
+    """Returns a simple python dict of used dockerImages by automations
+
+    :type content: ``str``
+    :param content: string representation for the /automation/search API result response
+
+    :return: dict as {"integration_id":"dockername"}
+    :rtype: dict
+
+    """
+    json_content = json.loads(content)
+    dockers = {}
+    for script in json_content[SCRIPTS]:
+        if (SCRIPT_TYPE not in script) or \
+                ((script[SCRIPT_TYPE] != PYTHON_SCRIPT) and (script[SCRIPT_TYPE] != POWERSHELL_SCRIPT)):
+            continue
+        else:
+            if DOCKER_IMAGE in script and script[DOCKER_IMAGE] in (None, ''):
+                docker_image = 'Default Image Name'
+            else:
+                docker_image = script[DOCKER_IMAGE]
+            dockers[script[SCRIPT_ID]] = docker_image
+
+    return dockers
+
+
+def merge_result(docker_list: dict, result_dict: dict = {}, max_entries_per_docker: int = 5) -> dict:
+    """Returns a python dict of the merge result
+
+    :type docker_list: ``dict``
+    :param docker_list: dictionary representation for the docker image used by integration or script
+
+    :type result_dict: ``dict``
+    :param result_dict: dictionary representation for the docker image and the integration/scripts belongs to it
+                       merge the current result to it
+
+    :type max_entries_per_docker: ``int``
+    :param max_entries_per_docker: max number of integration or script to show per docker image entry
+
+    :return: dict as {'docker_image':['integration/scripts', 'integration/scripts',...]}
+    :rtype: dict
+
+    """
+    result = result_dict or {}
+    for integration_script, docker_image in docker_list.items():
+        if integration_script in ['CommonServerUserPowerShell', 'CommonServerUserPython']:
+            continue
+        if docker_image in result:
+            if len(result[docker_image]) < max_entries_per_docker:
+                result[docker_image].append(integration_script)
+        else:
+            result[docker_image] = [integration_script]
+
+    return result
+
+
+def format_result_for_markdown(result_dict: dict) -> list:
+    result_output = []
+    for docker_image, integration_script in result_dict.items():
+        result_output.append({
+            'Docker Image': docker_image,
+            'Integrations/Automations': integration_script
+        })
+    return result_output
+
+
+''' COMMAND FUNCTION '''
+
+
+def get_used_dockers_images(export_to_context: bool = True) -> CommandResults:
+    md = None
+    active_docker_list_integration = {}
+    active_docker_list_automation = {}
+
+    ''' Examples for output: { 'demisto/python3:3.9.7.24076' : ['ListUsedDockerImage', 'VirusTotal',...]}'''
+    result_dict: Dict[str, List[str]] = {}
+
+    active_integration_instances = demisto.internalHttpRequest(POST_COMMAND, '/settings/integration/search',
+                                                               REQUEST_INTEGRATION_SEARCH_BODY)
```

Comment:
@wissamg please replace the const with the body dictionary for better readability.